app/frontend/src/components/forms/submission/StatusPanel.vue

```diff
@@ -224,6 +225,7 @@ export default {
   },
   methods: {
     ...mapActions('notifications', ['addNotification']),
+    ...mapActions('form', ['fetchSubmission']),
```

Comment:
I think this `fetchSubmission` is unused?

Reply:
Yea it's currently unused - doesn't need to be imported in.